aries-site/src/themes/aries.js

```diff
@@ -32,6 +30,23 @@ export const aries = deepMerge(hpe, {
       // background: undefined,
       border: { side: 'bottom' },
       color: 'text-strong',
+      extend: ({ column, sort, sortable }) => {
+        return `
+          ${sortable &&
+            sort &&
+            sort.property !== column &&
+            `
+              svg {
+                opacity: 0%;
+              }
+              :hover {
+                svg {
+                  opacity: 50%;
```

Comment:
```suggestion
                  opacity: 100%;
```
Designs seem to be calling for full opacity of `text-strong`: https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-Table?node-id=2390%3A13951

Reply:
Done. Thanks.